packages/common/src/ContractInteractor.ts

```diff
@@ -440,7 +466,8 @@ export class ContractInteractor {
 
   async getMaxViewableGasLimit (relayRequest: RelayRequest, maxViewableGasLimit: IntString): Promise<BN> {
     const maxViewableGasLimitBN = toBN(maxViewableGasLimit)
-    const gasPrice = toBN(relayRequest.relayData.gasPrice)
+    // TODO fix gasPrice
```

Comment:
Not sure there is anything to do here - maxFeePerGas serves its role for the max gas limit, doesn't it?

Reply:
Yes